client/src/components/player.js

```diff
@@ -27,8 +28,12 @@ function Player(player) {
     );
   }
 
+  if (player.wins === -1) {
```

Comment:
If we can't get w/l data then we shouldn't display anything besides name, should we?